src/api/gnosisProtocol/api.ts

```diff
@@ -32,7 +31,7 @@ const PROFILE_API_BASE_URL = getProfileUrl()
 
 const DEFAULT_HEADERS = {
   'Content-Type': 'application/json',
-  'X-AppId': APP_DATA_HASH.toString(),
+  'X-AppId': getAppDataHash(),
```

Comment:
This is a legacy parameter that has no effect AFAICT, and we should get rid of it

See [this thread](https://cowservices.slack.com/archives/C0375NV72SC/p1655734352461919) from 11 months ago

Reply:
I'm not sure we should do it! i think is nice to tell the backend :P 

@nlordell  @olgafetisova  this was created LOOOOONG ago! the idea is that we would signal in our API calls already what is the AppData. This is a legacy name, should be probably call `X-appData`

Why I think we shouldn't remove? 
Because, we can ignore in the backend, but already could give us some nice information about who is querying things. 
For example, which application, which UTM codes does it have, etc. 
it can be additional trace in the log, giving more info (we could see for example, people from a campaign asking for price and not sending the trade.

If this is something we don't want to have, I will delete it already in this PR. Nick/Olga, let me know if it's ok to leave or not